newlib/testsuite/newlib.stdio/tst-wc-printf.c

```diff
@@ -0,0 +1,22 @@
+#define _GNU_SOURCE 1
```

Comment:
Lack of copyright and license notice.

Reply:
> Lack of copyright and license notice.

Agreed. New files will need the author to insert a copyright statement and apply a license. For picolibc, please use the 3-clause BSD license as found at [opensource.org](https://opensource.org/licenses/BSD-3-Clause).

Files copied from elsewhere that are missing copyright and/or license information must have them added, along with
information that explains how those are correct.

Reply:
Added in latest commit